src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -514,12 +515,29 @@ protected function initializeContainer()
         $cache = new ConfigCache($this->getCacheDir().'/'.$class.'.php', $this->debug);
         $fresh = true;
         if (!$cache->isFresh()) {
-            $container = $this->buildContainer();
+            if ($this->debug) {
+                $collectedLogs = array();
+                $previousHandler = set_error_handler(function () use (&$collectedLogs, &$previousHandler) {
+                    $collectedLogs[] = $args = func_get_args();
+
+                    return $previousHandler;
```

Comment:
Doing this, logs are duplicated (regular process and collectedLogs), and so the debugToolbar display the wrong number of logs.
Exemple:
5 logs (4 from the boot process) + 4 boot logs = 9 logs (on container init)
1 Log + 4 boot logs = 5 logs (no new container init)